packages/waku/src/lib/middleware/rsc/worker-impl.ts

```diff
@@ -150,6 +168,19 @@ const shutdown = async () => {
   parentPort!.close();
 };
 
+const collectCssUrls = (mods: Set<ModuleNode>, styles: Map<string, string>) => {
+  for (const mod of mods) {
+    if (mod.ssrModule && mod.file && mod.id) {
+      if (mod.file.endsWith('.css') || /\?vue&type=style/.test(mod.id)) {
```

Comment:
why is there a `vue`

Reply:
This whole function is not used, I just copied it for an experiment from https://github.com/vitejs/vite/issues/2282#issuecomment-787049601

I'll remove it when I get this done!